### DISCUSSION TITLE
Problem for launch server with Docker compose

### DISCUSSION THREAD

**Discussion by LokazFR** [pg9182/northstar-dedicated#48] [Support]
Hello everybody,

I have a problem when I try to launch the server, I have this error message that appears:

_Status: Image is up to date for ghcr.io/pg9182/northstar-dedicated:1-tf2.0.11.0
docker: Error response from daemon: using mount program fuse-overlayfs: fuse: device not found, try 'modprobe fuse' first
fuse-overlayfs: cannot mount: No such file or directory_

Have you ever had this problem ? Searching the web, I can't really find any solutions, from what I understand, the command doesn't find my file path, even though it does exist.

Here is a example of file yml:

-----------------------------------------
version: "3.9"

services:
  northstar1:
    image: ghcr.io/pg9182/northstar-dedicated:1-tf2.0.11.0
    pull_policy: always
    environment:
      - NS_PORT=37015
      - NS_PORT_AUTH=8081
      - 'NS_SERVER_NAME=[TESTFRBX] TEST FR TEAM BX'
      - 'NS_SERVER_DESC=TEAM BX'
      - |
        NS_EXTRA_ARGUMENTS=
        +setplaylist private_match
        +net_compresspackets_minsize 64
        +net_compresspackets 1
        +spewlog_enable 0
        +sv_maxrate 127000
    volumes:
      - /home/root/Titanfall2:/mnt/titanfall:ro
    ports:
      - '37015:37015/udp'
      - '8081:8081/tcp'
    restart: always
------------------------------------------------------

I also tried to launch with the command "docker run" without success, same error.

I am on the latest Debian distribution (11.6), Docker and docker compose are well installed and up to date, I followed the instructions from the official site for their installation

Thank you all.

**Comment by BulletMemes** [comment]
`/home/root/Titanfall2:/mnt/titanfall:ro`

If you have the files in the root directory, /home/root/ is incorrect, it's only /root/.

Also please be sure that you're using capitals correctly, as it's case sensitive for the host side, if the folder has a capital then it's fine.

I would also recommend posting in the discord first before coming here, you'll get a response quicker.

**Comment by LokazFR** [comment]
Hello Bulletmemes,

it is a folder that I created and not the user folder of the root account I reassure you, I tried to start the "yml" file in another location, I have exactly the same error.

I also tried to deploy another container from another application, same same error, still related to fuseoverlaysfs!

Odd.....

I've read on the internet that there are quite a few issues with the latest versions of Docker compose and fuseoverlayfs, and they recommend testing with an older version like 1.29.2 which uses overlay2

Personally I am currently with version 2.16, can you confirm your version of docker compose?

I will reinstall my machine, start on a clean base and try again with an older version.

I am not a Docker expert, but having already practiced before, I have never encountered this type of problem.

Thank you.
